### PR TITLE
Assorted OpenGL fine tuning

### PR DESCRIPTION
**Pull request by michalsrb**
Hi, here are some small OpenGL optimizations I did while exploring your code.
I will send CLA to your email.

### CODE REVIEW

File: browser/src/vts-librenderer/renderer.cpp [hunk 1]
```diff
@@ -593,46 +604,51 @@ class RendererImpl
         widthPrev = heightPrev = antialiasingPrev = 0;
     }
 
-    void shaderUpdateAtmosphere(ShaderAtm *shader)
+    void updateAtmosphereBuffer()
     {
-        shader->bind();
-        float p[4] = {0,0,0,0};
-        shader->uniformVec4(shader->firstAtmUniLoc + 2, p);
+        struct ShaderAtm::AtmBlock atmBlock;
```
Comment (malytomas) [inline browser/src/vts-librenderer/renderer.cpp]: no need to repeat struct

File: browser/src/vts-libbrowser/include/vts-browser/math.hpp [hunk 2]
```diff
@@ -48,6 +48,8 @@ typedef Eigen::Matrix2d mat2;
 typedef Eigen::Matrix3d mat3;
 typedef Eigen::Matrix4d mat4;
 
+typedef Eigen::Matrix<unsigned short, 2, 1> vec2us;
```
[the remark this answers is not in the document]
Comment (michalsrb) [reply browser/src/vts-libbrowser/include/vts-browser/math.hpp]: Ok, makes sense.

File: browser/src/vts-librenderer/renderer.cpp (hunk 1 above)
Comment (michalsrb) [reply browser/src/vts-librenderer/renderer.cpp]: Oops, will change.

**Comment by michalsrb** [comment]
The requested modifications are in place now.

**Pull request merged**